package.json

```diff
@@ -18,7 +18,10 @@
     "lint": "eslint --ext .ts src/ test/",
     "pretest": "yarn check-types",
     "prepublishOnly": "yarn build",
-    "test": "nyc -r lcov -e .ts mocha -r ./.babel-register \"test/**/*.test.ts\" && nyc report"
+    "pretest": "npm run check-types",
+    "prepublishOnly": "npm run build-all",
```

Comment:
npm -> yarn. Also you have two pretest and prepublish tasks

Reply:
whoops, bad rebase